trin-history/src/lib.rs

```diff
@@ -113,3 +116,16 @@ pub fn spawn_history_network(
             .expect("failed to pause until ctrl-c");
     })
 }
+
+pub fn spawn_history_heartbeat(network: Arc<HistoryNetwork>) {
+    tokio::spawn(async move {
+        let mut heart_interval = interval(Duration::from_millis(30000));
+
+        loop {
+            let radius = network.overlay.data_radius();
+            let coverage_percent = *radius / U256::MAX * 100;
+            info!("status: coverage={coverage_percent:.2}%");
```

Comment:
Any thoughts on using structured logging here or whether that's the correct approach?

Reply:
I didn't find obvious google hits on "rust structured logging." What do you mean?

I imagine whatever it is will be easy to move to going forward, so I won't consider it a blocker for the PR.

Reply:
I can't believe that "rust structured logging" search doesn't take you straight here: https://docs.rs/log/latest/log/#structured-logging

Reply:
Ok, so the reason I'm not doing this before merge is that I want to use the float rounding display. That means presumably doing a format inside the structured value, which feels kind of awkward, like:
```rust
info!(radius = format!("{coverage_percent:.1}%"); "report");
```

So if someone comes up with a better idea, I'll include it in the next PR.